crypto/conf/conf_sap.c

```diff
@@ -60,10 +60,6 @@ int openssl_config_int(const OPENSSL_INIT_SETTINGS *settings)
 #endif
 
     OPENSSL_load_builtin_modules();
-#ifndef OPENSSL_NO_ENGINE
-    /* Need to load ENGINEs */
-    ENGINE_load_builtin_engines();
-#endif
```

Comment:
Why is this no longer required?

Reply:
It's the wrong spot to call `ENGINE_load_builtin_engines()`, as this code is called by `OPENSSL_init_crypto()` and `ENGINE_load_builtin_engines()` also calls `OPENSSL_init_crypto()`.